client/src/apis/channels.ts

```diff
@@ -1,6 +1,8 @@
 import instance from ".";
 
-export default async function fetchChannels() {
+const fetchChannels = async () => {
```

Comment:
Lets do named export for all api methods. Because other api files will have multiple methods in single file. So, we can maintain consistency by doing named export for all methods
```suggestion
export const fetchChannels = async () => {
```